config_template.cfg

```diff
@@ -1434,9 +1434,10 @@ SOLUTION_ADJ_FILENAME= solution_adj.dat
 TABULAR_FORMAT= TECPLOT
 %
 % Files to output 
-% Possible formats : (TECPLOT, TECPLOT_BINARY, SURFACE_TECPLOT,
-%  SURFACE_TECPLOT_BINARY, CSV, SURFACE_CSV, PARAVIEW, PARAVIEW_BINARY, SURFACE_PARAVIEW, 
-%  SURFACE_PARAVIEW_BINARY, MESH, RESTART_BINARY, RESTART_ASCII, CGNS, STL)
+% Possible formats : (TECPLOT, TECPLOT_ASCII, SURFACE_TECPLOT, SURFACE_TECPLOT_ASCII,
+%  CSV, SURFACE_CSV, PARAVIEW, PARAVIEW_ASCII, PARAVIEW_LEGACY, SURFACE_PARAVIEW,
+%  SURFACE_PARAVIEW_ASCII, SURFACE_PARAVIEW_LEGACY, PARAVIEW_MULTIBLOCK, RESTART,
+%  RESTART_ASCII, CGNS, STL, STL_BINARY, MESH)
```

Comment:
:+1: Do we have a "MESH" option?

Reply:
https://github.com/su2code/SU2/blob/develop/Common/include/option_structure.hpp line 1631

Reply:
Indeed but what matters are the strings from 1642 to 1659, the enums are for internal use.

Reply:
Okay, but "MESH" is indeed an output option, see https://github.com/su2code/SU2/blob/develop/SU2_CFD/src/output/COutput.cpp line 388. Maybe the pairing in option_structure is missing?

Reply:
It is only used internally in SU2_DEF I think, it does not need to be selected by users.

Reply:
Good. So shall be removed from the config_template.